### DISCUSSION TITLE
apakah masih layak di tahun 2024?

### DISCUSSION THREAD

**Discussion by rexyb9** [deaafrizal/forum#57] [All About Hardware ( PC, Laptop etc...)]
Processor Intel Core i3 12100F
VGA Asus NVIDIA GT 1030 2GB DDR5
PSU Deepcool 400w 80+
Motherboard Gigabyte h610m LGA 1700 DDR4 
SSD NVMe 128GB Transcend
Harddisk SATA 2.5" 500GB
Monitor 24inch
Casing Aerocool Talon
Keyboard Votre
Mouse Votre

**Comment by redoamain** [comment]
layak bang
